backend/OpenAPI.yaml

```diff
@@ -20,61 +20,38 @@ paths:
         content:
           application/json:
             schema:
-              type: object
-              required:
-                - email
-                - password
-              properties:
-                email:
-                  type: string
-                  format: email
-                  example: "student@utoronto.ca"
-                  description: "Must be a valid University of Toronto email (utoronto.ca or mail.utoronto.ca)."
-                password:
-                  type: string
-                  format: password
-                  example: "P@ssword123"
-                  description: "Must be at least 8 characters, contain 1 uppercase letter, 1 number."
+              $ref: "#/components/schemas/SignUpPostRequest"
       responses:
         "201":
           description: User created successfully
           content:
             application/json:
               schema:
-                type: object
-                required:
-                  - user_id
-                properties:
-                  user_id:
-                    type: integer
-                    example: 42
-                  message:
-                    type: string
-                    example: "User registered successfully."
+                $ref: "#/components/schemas/SignUpPostResponse"
         "400":
-          description: Bad Request - Invalid email or password format
+          description: Bad Request - Invalid request format
```

Comment:
I think you got this flipped with `422`. `400` should be for in valid email/password while `422` is for invalid request.